src/mudlet.pro

```diff
@@ -108,8 +108,16 @@ $$system(git --version, blob, GIT_STATUS)
 isEqual(GIT_STATUS, 0) {
     # Git command appeared to work - so get a description of the current
     # repository state
-    GIT_DESCRIPTION=$$system(git describe --all --dirty --broken)
-    GIT_COMMITHASH=$$system(git rev-parse --short HEAD)
+    GIT_DESCRIPTION = $$(GITHUB_PR_BRANCH)
+    isEmpty($$(GITHUB_PR_BRANCH)) {
+        GIT_DESCRIPTION=$$system(git describe --all --dirty --broken)
+    }
+    isEmpty($$GITHUB_SHA) {
```

Comment:
You need `(`...`)` around an *environmental variable* if you want to access it within QMake:
```suggestion
    isEmpty($$(GITHUB_SHA)) {
```